packages/react/src/runtime/ax.ts

```diff
@@ -1,64 +1,80 @@
-const UNDERSCORE_UNICODE = 95;
-
 /**
```

Comment:
Nice refactoring! Can you please run the benchmark script, that is in the root package.json? I am interested to see the perf improvement before vs after. 
Thank you!

Reply:
Further exploration: https://www.youtube.com/watch?v=SWfujKr4-tM